### ISSUE TITLE
`count` is marked as undefined variable

### ISSUE THREAD

**Issue by zerkms** [hashicorp/vscode-terraform#220]
```hcl
resource "aws_vpc" "vpc" {
  cidr_block = "10.10.0.0/16"
}

resource "aws_subnet" "subnet" {
  count = 3

  vpc_id = aws_vpc.vpc.id
  cidr_block = "10.10.${count.index + 1}.0/24"
  availability_zone_id = "apse2-az${count.index + 1}"
}
```

In the following HCL `count` (both instances) is marked as `There is no variable named "count".`

**Comment by Vermyndax** [comment]
Having this issue on language server 0.9 too.

**Comment by zachovich** [comment]
Same here on 0.0.9

**Comment by audricganser** [comment]
@zachovich
Not sure if this is your issue but from reading the Terraform docs it looks like `count` is not a supported argument for `aws_iam_role_policy_attachment`.
https://www.terraform.io/docs/providers/aws/r/iam_role_policy_attachment.html#argument-reference

**Comment by zerkms** [comment]
@audricganser https://www.terraform.io/docs/configuration/resources.html#count-multiple-resource-instances-by-count

**Comment by audricganser** [comment]
I was just reading about meta-arguments and yes I suppose it should be able to handle `count`.

**Comment by borikoss** [comment]
having the same issue after tf upgrade to the version 12. Any progress here for a solution? Thanks!

**Comment by DJAlPee** [comment]
There is already a Pull Request pending, which will fix this issue in the language server: https://github.com/juliosueiras/terraform-lsp/pull/43

**Comment by borikoss** [comment]
@DJAlPee great, thx :) It seems as the PR has been merged yesterday

**Comment by johnnyplaydrums** [comment]
Just wondering are folks still experiencing this issue? I am still getting the `count` error. Do I need to update something or is the fix still pending? Thanks for the help 🙂

**Comment by wcamarao** [comment]
@mauve any chance you could release a new version including this change from lsp? 🙏

**Comment by paultyng** [comment]
We just released [v2.0.0-rc.1](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) of the extension. The main features include:

* Added syntax support for 0.12
* Added [terraform-ls](https://github.com/hashicorp/terraform-ls) usage by default (currently on 0.3.0, which offers basic provider code completion)

You can find additional information and specifics in the [release notes](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) and [CHANGELOG](https://github.com/hashicorp/vscode-terraform/blob/master/CHANGELOG.md).

With this release we expect that many of the prior issues and PRs are no longer relevant or have been addressed, and are therefore being closed. If you feel the action taken on an issue or PR is in error, please comment as such and we can figure out the appropriate way to address it.

We plan to add the final 2.0.0 release to the marketplace soon, but are actively seeking your feedback now on the release candidates. You can download the .vsix from the releases page and [manually install it in VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix) to try it out.
